src/js/utils/routesController.js

```diff
@@ -60,4 +60,43 @@ module.exports = class RoutesController {
       });
     }
   };
+
+  updateItem = async (req, reply) => {
+    try {
+      const { id } = req.params;
+      const item = await this.model.findOne({
+        where: { id },
+      });
+      let updatedItem;
+
+      if (!item)
+        throw new Error(
+          `${this.modelName} with the specified ID does not exist`
+        );
+
+      const [isUpdated] = await this.model.update(req.body, {
```

Comment:
can replace where to the line 77, like this:

const [isUpdated] = await this.model.update(req.body, {where: { id } });